dts/nios2/nios2f.dtsi

```diff
@@ -50,6 +50,7 @@
 			#address-cells = <1>;
 			#size-cells = <0>;
 			reg = <0x100200 0x400>;
+			interrupts = <4 10>;
```

Comment:
Should `interrupts` be made a required property, since we've eliminated the non-DTS way of providing this information?